src/local-ssh/pipeSession.ts

```diff
@@ -0,0 +1,192 @@
+import { CancellationToken, PromiseCompletionSource, SessionRequestFailureMessage, SessionRequestMessage, SessionRequestSuccessMessage, SshChannel, SshChannelClosedEventArgs, SshChannelError, SshChannelOpeningEventArgs, SshMessage, SshRequestEventArgs, SshSession, SshSessionClosedEventArgs, SshTraceEventIds, TraceLevel } from "@microsoft/dev-tunnels-ssh";
```

Comment:
is it a temporary copy from the upstream? It would nice to have a comment? Or rather fork and use our version?

Reply:
Jean Pierre's upstream PR https://github.com/microsoft/dev-tunnels-ssh/pull/65 is approved, it's easier to patch this way. We can delete this file after upstream release